lib/Ed25519KeyPair.js

```diff
@@ -33,6 +33,8 @@ class Ed25519KeyPair extends LDKeyPair {
 
   /**
    * @param [options] {object} See LDKeyPair docstring for full list
+   * @param [options.seed] {(Uint8Array|Buffer)} a 32-byte seed for a
```

Comment:
Very minor, but just for future reference - the `()` is not needed, just ` {Uint8Array|Buffer}` works.